pkg/pipeline/transform/transform_network.go

```diff
@@ -123,6 +123,24 @@ func (n *Network) Transform(inputEntry config.GenericMap) (config.GenericMap, bo
 					outputEntry[rule.Output+"_HostName"] = kubeInfo.HostName
 				}
 			}
+		case api.OpAddKubernetesInfra:
+			srcKubeInfo, err := kubernetes.Data.GetInfo(fmt.Sprintf("%s", outputEntry["SrcAddr"]))
+			if err != nil {
+				logrus.WithError(err).Tracef("can't find kubernetes info for IP %v", outputEntry["SrcAddr"])
+				continue
+			}
+			dstKubeInfo, err := kubernetes.Data.GetInfo(fmt.Sprintf("%s", outputEntry["DstAddr"]))
+			if err != nil {
+				logrus.WithError(err).Tracef("can't find kubernetes info for IP %v", outputEntry["DstAddr"])
+				continue
+			}
+
+			if objectIsApp(*srcKubeInfo, rule.Parameters) || objectIsApp(*dstKubeInfo, rule.Parameters) {
```

Comment:
We should move `GetInfo` calls into `objectIsApp` function. 

Let's say `srcKubeInfo` is app and `dstKubeInfo` is not available; we will miss `K8S_FlowType` whereas it could be identified as app since we know the source.

Also, when both `src` and `dst` are not identified, should we consider these as `app` ?

Reply:
Moved the GetInfo in obectIsApp thanks.

For now, every flow that is not tagged as app is tagged as infra.